card/mod/edit/set/all/edit_in_place.rb

```diff
@@ -16,10 +16,6 @@
      "<div class='col-sm-11'>#{render_editable}</div>"]
   end
 
-  view :editable, perms: :update, tags: :unknown_ok, wrap: :slot do
```

Comment:
this view is used in the view above

Reply:
Doh, missed that one. I'll just use `render_menued; edit: content_inline`, so we don't have to have separate views for different editing patterns.